clinical/src/test/java/com/hartwig/actin/clinical/curation/CurationDatabaseValidatorTest.java

```diff
@@ -0,0 +1,89 @@
+package com.hartwig.actin.clinical.curation;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import com.google.common.collect.Lists;
+import com.google.common.collect.Sets;
+import com.hartwig.actin.clinical.curation.config.IntoleranceConfig;
+import com.hartwig.actin.clinical.curation.config.NonOncologicalHistoryConfig;
+import com.hartwig.actin.clinical.curation.config.PrimaryTumorConfig;
+import com.hartwig.actin.clinical.curation.config.SecondPrimaryConfig;
+import com.hartwig.actin.clinical.curation.config.TestCurationConfigFactory;
+import com.hartwig.actin.clinical.datamodel.TestPriorOtherConditionFactory;
+import com.hartwig.actin.clinical.datamodel.TestPriorSecondPrimaryFactory;
+import com.hartwig.actin.doid.DoidModel;
+import com.hartwig.actin.doid.TestDoidModelFactory;
+
+import org.junit.Test;
+
+public class CurationDatabaseValidatorTest {
+
+    @Test
+    public void doesNotCrashOnTestCurationDatabase() {
+        CurationDatabaseValidator validator = TestCurationFactory.createMinimalTestCurationDatabaseValidator();
+
+        validator.validate(TestCurationFactory.createTestCurationDatabase());
+    }
+
+    @Test
+    public void canIdentifyInvalidPrimaryTumorConfigs() {
```

Comment:
I usually try to include "should" at the start of the test name instead of "can" or "does". Do we have/want a convention for test names?

Reply:
There is no convention for test names other than that they should represent what the test tests for. 

I can see why "should" is better in this case, will update